api/src/main/resources/workbench.yaml

```diff
@@ -754,7 +754,6 @@ paths:
           description: specifies which cdr version
         - in: body
           name: request
-          type: integer
           required: true
           description: size of the participant cohort statuses sample
           schema:
```

Comment:
The description here is wrong. @freemabd1 may want to fix in a follow-up.

Reply:
will do

Reply:
Eh, may as well fix while I'm touching this line. Done.

Reply:
thanks